### PR TITLE
Code + assets for frontend

### PR DESCRIPTION
**Pull request by seanmtracey**
Now interacts with the live data-for endpoint instead of the fake one.

### CODE REVIEW

File: client/src/main.js [hunk 1]
```diff
@@ -1 +1,30 @@
-'use strict'; // eslint-disable-line strict
+/* Global Math, console */
```
Comment (JakeChampion) [inline client/src/main.js]: I think you could do `/* eslint-env browser */` or something along those lines

File: client/src/main.js [hunk 2]
```diff
@@ -1 +1,30 @@
...
+			border-radius: 5px;
+		    box-shadow: 0 0 5px black;
+		}
+
+	`;
+
+	document.head.appendChild(style);
+	document.body.appendChild(holder);
+
+}());
```
Comment (JakeChampion) [inline client/src/main.js]: I don't think this needs to be in a IIFE.

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+const gulp = require('gulp');
+const preprocess = require('gulp-preprocess');
+const uglify = require('gulp-uglify');
+const insert = require('gulp-insert');
+const env = require('dotenv').config();
```
Comment (JakeChampion) [inline gulpfile.js:6]: This could be 
`require('dotenv').load();`

File: server/routes/clientData.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+const debug = require('debug')('perf-widget:lib:clientData');
+const dataFor = require('../lib/dataFor');
+/*const dataFor = function (){
```
Comment (JakeChampion) [inline server/routes/clientData.js]: Could we remove this now?

File: client/src/main.js (hunk 2 above)
Comment (seanmtracey) [inline client/src/main.js]: Doesn't need to be in at all. This is the old implementation. Will delete.

File: server/routes/clientData.js (hunk 4 above)
Comment (seanmtracey) [inline server/routes/clientData.js]: Sure, I just wanted to make sure it was somewhere in the commit history while we were still putting bits together

File: gulpfile.js (hunk 3 above)
Comment (seanmtracey) [inline gulpfile.js:6]: Is there a discernible difference?

Comment (JakeChampion) [inline gulpfile.js:6]: No idea :-)

**Pull request merged**